src/app/src/main/kotlin/org/vaccineimpact/api/app/ChunkedFileCache.kt

```diff
@@ -34,10 +34,6 @@ class ChunkedFileCache(private val flushInterval: Long = TimeUnit.HOURS.toMillis
         {
             memoryCache[item.uniqueIdentifier] = MutablePair(item, System.currentTimeMillis())
         }
-        else
```

Comment:
@EmmaLRussell you alerted me to this being completely obsolete - the uploadedChunks map hasn't been written to yet at this point.